ui/components/app/snaps/snap-ui-markdown/snap-ui-markdown.js

```diff
@@ -17,7 +17,7 @@ import {
 } from '../../../component-library';
 import SnapLinkWarning from '../snap-link-warning';
 
-const Paragraph = (props) => (
+export const Paragraph = (props) => (
```

Comment:
Why is this exported now? Feels like an implementation detail for the component, especially with that name

Reply:
Unintended, I was about to do something 😅